packages/core/src/types/module-builder.ts

```diff
@@ -204,6 +360,26 @@ export interface IgnitionModuleBuilder {
     options?: ReadEventArgumentOptions
   ): ReadEventArgumentFuture;
 
+  /**
+   * Sends a transaction.
```

Comment:
We should expand on this to explain it allows you to send an arbitrary transaction, so you can transfer eth or invoke with raw data.